examples/gsutil/cat.rs

```diff
@@ -25,14 +24,14 @@ last numbytes of the object."
     url: url::Url,
 }
 
-pub(crate) fn cmd(ctx: &util::RequestContext, args: Args) -> Result<(), Error> {
+pub(crate) fn cmd(ctx: &util::RequestContext, args: Args) -> Result<(), anyhow::Error> {
     let oid = util::gs_url_to_object_id(&args.url)?;
 
     let mut download_req = Object::download(
         &(
             oid.bucket(),
             oid.object()
-                .ok_or_else(|| failure::format_err!("invalid object name specified"))?,
+                .ok_or_else(|| anyhow::anyhow!("invalid object name specified"))?,
```

Comment:
If you prefer, this can be written more concisely as

```rust
        oid.object().context("invalid object name specified")?,
```

with `anyhow::Context` in scope. Similar for all the other uses of ok\_or\_else below.

Reply:
Ahh right, dunno why it was like that to begin with, thanks for pointing this out. :slightly_smiling_face: